release-02/code/flag_tool.rb

```diff
@@ -0,0 +1,38 @@
+require 'sequel'
+
+helpText = "ITU-Minitwit Tweet Flagging Tool\n\n" +
+            "Usage:\n" +
+            "  flag_tool <tweet_id>...\n" +
+            "  flag_tool -i\n" +
+            "  flag_tool -h\n" +
+            "Options:\n" +
+            "-h            Show this screen.\n" +
+            "-i            Dump all tweets and authors to STDOUT.\n";
+
+
+def printTweet(tweet_id, user_id, tweet_text, flagged)
+    begin
+        printf "%s,%s,%s,%s\n", tweet_id, user_id, tweet_text, flagged.nil? ? 0 : flagged
+    end
+end
+
+if ARGV.length == 1 && ARGV[0].eql?("-h") then
+    puts helpText
+    exit(true)
+end
+DB = Sequel.sqlite('/tmp/minitwit.db')
```

Comment:
We need to make sure that this is the place where we will place our database. Maybe we can setup environment variables to determine the location of the database from outside the script. Nothing to worry about yet.